index.html

```diff
@@ -1192,33 +1192,31 @@ <h3>
       </p>
 
       <p>
-        A DID method specification should have a section dedicated to personal
-        data covering the extent to
+        DID method specifications are encouraged to have a section dedicated to
+        personal data, covering the extent to
         which information published on the corresponding ledger can be updated
         or deleted.
-        It should provide specific instructions of how to do so, if it is
-        possible.
+        Such a section could provide specific instructions of how to do so,
+        if it is possible.
         Otherwise, it should clearly state that it is not possible.
       </p>
 
       <p>
         A DID method should avoid "phone home" or tracking characteristics that
-        would
-        permit tracing of a user in manner not understood or authorized by the
-        user
-        by some third party.
+        would permit tracing of a user by a third party
+        in manner not understood or authorized by the user.
```

Comment:
```suggestion
        would enable a third party to track a user without the user's consent,
        in a manner the user may not understand.
```

Reply:
I applied the similar earlier change https://github.com/w3c/did-imp-guide/pull/37#discussion_r759438237 (permit -> enable, tracing -> tracking, consent).
I like that this change uses the active voice.
But is it okay to not mention "authorized" here - i.e. that it is implied by "consent"?
Could use this change but change "consent" to "consent/authorization"?

Reply:
> [@clehner] Could use this change but change "consent" to "consent/authorization"?

Sure, `s/consent/authorization/` applied above.

Reply:
@TallTed
This change suggestion (diff) doesn't apply cleanly now, due to the changes in https://github.com/w3c/did-imp-guide/pull/37#discussion_r759438237 (26b848ba9e49008e1fc8860b08a46f4fc6404eaa).
I see your newer change suggestion on this sentence as updated, here: https://github.com/w3c/did-imp-guide/pull/37#discussion_r762364890.
The newer suggested phrasing is different from the one here.
Does the newer change suggestion supersede this one?